modules/dns/variables.tf

```diff
@@ -1,11 +1,20 @@
 variable "vpc_id" {
   description = "VPC ID where GraphDB will be deployed"
   type        = string
+
+  validation {
+    condition     = can(regex("^[a-zA-Z0-9-]+$", var.vpc_id))
```

Comment:
Aren't VPC IDs starting with `vpc-`? While the check is technically correct, you can make it more strict by specifying the prefix.

Reply:
Good point. I have added that to validation, as well as everywhere else that there is this ``variable "vpc_id"``